roles/setup/vars/RedHat-8.yml

```diff
@@ -1,5 +1,5 @@
-additional_system_packages:
-  - python39-pip
+system_pip_package: python39-pip
```

Comment:
we should generalize this pattern for Debian and RHEL9 too

Set
`additional_system_packages` to empty list in the defaults

and set
`system_pip_package: python3-pip` for both vars/Debian.yml and vars/RedHat.yml